pkg/clusterresource/builder.go

```diff
@@ -106,6 +106,9 @@ type Builder struct {
 
 	// MachineNetwork is the subnet to use for the cluster's machine network.
 	MachineNetwork string
+
+	// SkipMachinePools should be true if you do not want to manage MachineSets in-cluster once it is installed.
```

Comment:
The term "in-cluster" is confusing here.
```suggestion
	// SkipMachinePools should be true if you do not want to Hive to manage MachineSets once the cluster is installed.
```